DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h

```diff
@@ -36,9 +36,11 @@ class Diagnostic
   int getFrequency(ULong64_t pattern);                                                    // Get frequency
   int getFrequencyROW() { return getFrequency(0); }                                       // Readout window frequency
   int getFrequencyEmptyCrate(int crate) { return getFrequency(getEmptyCrateKey(crate)); } // empty crate frequency
+  int fillNoisy(int channel, int frequency = 1) { return fill(getNoisyChannel(channel), frequency); }
   int fillROW() { return fill(0); }
   int fillEmptyCrate(int crate, int frequency = 1) { return fill(getEmptyCrateKey(crate), frequency); }
   ULong64_t getEmptyCrateKey(int crate);
+  ULong64_t getNoisyChannel(int channel);
```

Comment:
isn't this const method?

Reply:
Yes, I will fix it (the same for getEmptyrateKey)